lms/tests/test_exercise_unit_tests.py

```diff
@@ -14,21 +14,37 @@ def foo(bar=None):
 '''
 
 EXERCISE_TESTS = os.path.join(conftest.SAMPLES_DIR, 'student_test_code.py')
+INVALID_EXERCISE_TESTS = os.path.join(
```

Comment:
I think pathlib can make more sense here, but ayw

Reply:
But I only need the file path here, not the module itself